partner_survey/readme/CONTRIBUTORS.md

```diff
@@ -1,8 +1,10 @@
-- Javier Iniesta \<<javieria@antiun.com>\>
-- Damien Crier \<<damien.crier@camptocamp.com>\>
-- Akim Juillerat \<<akim.juillerat@camptocamp.com>\>
-- [Tecnativa](https://www.tecnativa.com):
-  - Rafael Blasco
-  - Antonio Espinosa
-  - Ernesto Tejeda
-  - Stefan Ungureanu
+* Javier Iniesta <javieria@antiun.com>
+* Damien Crier <damien.crier@camptocamp.com>
+* Akim Juillerat <akim.juillerat@camptocamp.com>
```

Comment:
no need to update the syntax of these lines

Reply:
some minor changes:
- follow to this [guide](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0#technical-method-to-migrate-a-module-from-160-to-170-branch) to update the the precommit message and PR title

Reply:
> no need to update the syntax of these lines, just append your own email

I just append my own email but pre-commit fix those syntax

Reply:
```- Javier Iniesta \<<javieria@antiun.com>\>
- Damien Crier \<<damien.crier@camptocamp.com>\>
- Akim Juillerat \<<akim.juillerat@camptocamp.com>\>
- [Tecnativa](https://www.tecnativa.com):
  - Rafael Blasco
  - Antonio Espinosa
  - Ernesto Tejeda
  - Stefan Ungureanu
- Chau Le \<<chaulb@trobz.com>\>
```

Reply:
also, please fix the pre-commit message according to this [guide](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0#technical-method-to-migrate-a-module-from-160-to-170-branch)